app/models/lookup_value.rb

```diff
@@ -14,6 +14,7 @@ class LookupValue < ActiveRecord::Base
   validate :validate_list, :validate_regexp, :ensure_fqdn_exists, :ensure_hostgroup_exists
 
   attr_accessor :host_or_hostgroup
+  attr_writer :managed_id, :hostgroup_id
```

Comment:
host_common.rb:49 - lookup_values.build tries to set managed_id or hostgroup_id for host/hostgroup, we let it do it and ignore it.